src/Model/Page.php

```diff
@@ -1005,7 +1005,8 @@ protected function formatObject(
         // --------------------------------------------------------------------------
 
         //  Owner
-        $modifiedBy                     = (int) $oObj->modified_by;
+        $modifiedBy = (int) is_object($oObj->modified_by) ? $oObj->modified_by->id : $oObj->modified_by;
```

Comment:
Does this do what you expect? Is it casting the output of `is_object`, or the result of the ternary?

Reply:
@hellopablo good catch, is was casting the return of `is_object` to an int, then checking that value's equivalence to true. I've pushed a fix.